packages/webcert/src/feature/certificate/Inputs/UeSickLeavePeriod/UeSickLeavePeriod.tsx

```diff
@@ -30,6 +31,13 @@ const AccordionStyles = css`
   flex: 0 0 100%;
 `
 
+const TextInputStyles = css`
+  width: 40px;
+  padding: 4px 4px;
+  height: 35px;
+  text-align: center;
+`
```

Comment:
Since you added `className` prop to `TextInput` you could use a styled component mixin instead
```suggestion
const MyTextInput = styled(TextInput)`
  width: 40px;
  padding: 4px 4px;
  height: 35px;
  text-align: center;
`
```